build.sbt

```diff
@@ -1,38 +1,20 @@
-name := "messagepackKafkaSerde"
+name := "kafka_serde"
+version := "1.0.0"
+organization := "com.punchcyber"
+crossScalaVersions := Seq("2.12.10", "2.13.3")
 
-version := "0.1"
-
-scalaVersion := "2.13.1"
-
-assemblyJarName in assembly := name.value + "-" + version.value + ".jar"
-
-resolvers ++= Seq(
-    "clojars"           at "https://clojars.org/repo/"
-)
+credentials += Credentials(Path.userHome / ".sbt" / ".credentials")
+publishMavenStyle := true
+publishTo := {
+  val azureArtifacts =
+    "https://pkgs.dev.azure.com/Punch-Research/2ad94760-8a08-4dea-b480-6ff857d18c06/_packaging/etl-utilities-release/maven/v1"
```

Comment:
This should be `https://pkgs.dev.azure.com/Punch-Research/2ad94760-8a08-4dea-b480-6ff857d18c06/_packaging/`, and let the parts below be:

```
  if (isSnapshot.value)
    Some("snapshots" at azureArtifacts + "etl-utilities-release/maven/v1")
  else
    Some("releases" at azureArtifacts + "etl-utilities-snapshot/maven/v1")
```

You could create a set of artifact repos for kafka-serde, but it probably makes sense to keep them at the project name level.